### PR TITLE
RPET-981: Fix dependency errors

### PR DESCRIPTION
**Pull request by Dizrupto**
# Description

Fix functional test: 403 error access denied on CCD

Fixes https://tools.hmcts.net/jira/browse/RPET-981

### CODE REVIEW

File: charts/div-da/values.yaml [hunk 1]
```diff
@@ -22,8 +22,8 @@ nodejs:
     WEBCHAT_BUTTON_SERVICE_CLOSED: 20199488815cc1a89e0861d5.73103009
     REFORM_ENVIRONMENT: "{{ .Values.global.environment }}"
 
-    BASE_URL: "https://decree-absolute-aks.{{ .Values.global.environment }}.platform.hmcts.net"
-    PUBLIC_HOSTNAME: "https://decree-absolute-aks.{{ .Values.global.environment }}.platform.hmcts.net"
+    BASE_URL: "https://div-da-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
```
Comment (Dizrupto) [inline charts/div-da/values.yaml:25]: Changing the URLs didn't do anything on master build actually

File: test/functional/helpers/idamHelper.js [hunk 2]
```diff
@@ -23,7 +23,7 @@ class IdamHelper extends Helper {
       idamArgs.testEmail = testEmail;
       idamArgs.testPassword = testPassword;
       idamArgs.testGroupCode = 'citizens';
-      idamArgs.roles = [{ code: 'citizen' }, { code: 'caseworker-divorce-courtadmin_beta' }];
+      idamArgs.roles = [{ code: 'caseworker-divorce-courtadmin_beta' }, { code: 'citizen' }];
```
Comment (Dizrupto) [inline test/functional/helpers/idamHelper.js:26]: it seems IDAM has changed something in the way they assign roles. It seems that changing the order changes the priority of roles assigned to the user.

**Review by yarpo — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -126,7 +126,8 @@
     "netmask": "^2.0.1",
     "diff": "^3.5.0",
     "hosted-git-info": "3.0.8",
-    "url-parse": "1.5.0"
+    "url-parse": "1.5.0",
```
Comment (yarpo) [inline package.json:129]: `>=` or `^` ?

File: package.json [hunk 4]
```diff
@@ -126,7 +126,8 @@
...
+    "postcss": "8.2.10"
```
Comment (yarpo) [inline package.json:130]: as above?

File: package.json [hunk 5]
```diff
@@ -126,7 +126,8 @@
     "netmask": "^2.0.1",
     "diff": "^3.5.0",
     "hosted-git-info": "3.0.8",
```
Comment (yarpo) [inline package.json:128]: as below :)

File: package.json (hunk 3 above)
Comment (Dizrupto) [reply package.json:129]: sure

**Review by HarryH96 — APPROVED** [review]

**Pull request merged**